concent_api/gatekeeper/views.py

```diff
@@ -25,12 +25,14 @@
 @csrf_exempt
 @require_POST
 def upload(request):
+    logger.debug("Upload request received.")
     if request.content_type != 'application/x-www-form-urlencoded':
         return gatekeeper_access_denied_response('Unsupported content type.')
 
-    response = parse_headers(request)
+    path_to_file = request.get_full_path().partition(reverse('gatekeeper:upload'))[2]
+    response = parse_headers(request, path_to_file)
     if response is not None:
-        logger.warning(response.content.decode())
+        logger.info(response.content.decode())
```

Comment:
This change should be in a separate commit. The description of your commit does not mention it. Please don't mix in random changes to unrelated commits.

Reply:
I separated this change to commit "Switch logger.warning with access_denied_response to legger.info"